.env.example

```diff
@@ -0,0 +1,6 @@
+TOKEN=
```

Comment:
I'll make a shell script for these too eventually so we can just go `./scripts/make-env-files`